test/isEqual.js

```diff
@@ -516,6 +516,28 @@ describe('isEqual', function() {
     }
   });
 
+  it('should distinguish between map keys and values', function() {
+    if (Map) {
+      var map1 = new Map,
+          map2 = new Map;
+
+      map1.set('a', 'b');
```

Comment:
Would swap the `a` and `b` for `KEY`  and `VALUE` just so future readers will be able to understand the point :)